MusicHatSkill/src/main/kotlin/furhatos/app/musichatskill/SpotifyApiHandler.kt

```diff
@@ -7,8 +7,8 @@ import com.adamratzman.spotify.spotifyAppApi
 import com.adamratzman.spotify.utils.Market
 
 class SpotifyApiHandler {
-    private val clientID = ""
-    private val clientSecret = ""
```

Comment:
We should not push the credentials to a public repo. Let's keep these empty in the repo and update them only in our own branches.
In the future, don't git add this file after adding the credentials.